voc/python/ast.py

```diff
@@ -463,7 +463,12 @@ def visit_For(self, node):
         )
 
         loop = START_LOOP()
-
+        self.context.add_opcodes(
+            java.New('org/python/types/Bool'),
+            JavaOpcodes.ICONST_1(),
+            java.Init('org/python/types/Bool', 'Z'),
+            ASTORE_name('#for-orelse-%x' % id(loop))
```

Comment:
Given that this variable won't ever be accessed by the user, it would be nice to use a primitive, rather than a full object. using ISTORE_0 and ISTORE_1 to store an integer constant instead of a boolean object will be much faster.